src/Cortside.RestApiClient/PolicyBuilderExtensions.cs

```diff
@@ -0,0 +1,110 @@
+#pragma warning disable RCS1224 // Make method an extension method.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using Cortside.Common.Validation;
+using Polly;
+using Polly.Contrib.WaitAndRetry;
+using RestSharp;
+
+namespace Cortside.RestApiClient {
+    /// <summary>
+    /// Contains convenience methods for configuring policies to handle conditions typically representing transient faults when making <see cref="RestClient"/> requests.
+    /// </summary>
+    public static class PolicyBuilderExtensions {
+        private static readonly Func<RestResponse, bool> TransientHttpStatusCodePredicate = (response) => {
+            return (int)response.StatusCode >= 500 || response.StatusCode == HttpStatusCode.RequestTimeout;
+        };
+
+        public static IEnumerable<TimeSpan> Jitter(int firstRetrySeconds, int retryCount) {
+            return Backoff.DecorrelatedJitterBackoffV2(medianFirstRetryDelay: TimeSpan.FromSeconds(firstRetrySeconds), retryCount: retryCount);
```

Comment:
nice - is Backoff something new that comes with Polly now?

Reply:
No, but it comes from Polly.Contrib.WaitAndRetry which is called out by Polly proper.